README.md

```diff
@@ -2,181 +2,90 @@
-processing these blocks and publishing them to the other node. If you have
-issues, try restarting the beacon nodes to ensure they have the same genesis
-time. Alternatively, raise an issue and include your terminal output.
-
-## Further Reading
-
-- [About Lighthouse](docs/lighthouse.md): Goals, Ideology and Ethos surrounding
-this implementation.
-- [What is Ethereum Serenity](docs/serenity.md): an introduction to Ethereum Serenity.
-- [Lighthouse Technical Documentation](http://lighthouse-docs.sigmaprime.io/): The Rust generated documentation, updated regularly.
+Code documentation is generated via `cargo doc` and hosted at
+[lighthouse-docs.sigmaprime.io](http://lighthouse-docs.sigmaprime.io/).
 
 If you'd like some background on Sigma Prime, please see the [Lighthouse Update
-\#00](https://lighthouse.sigmaprime.io/update-00.html) blog post or the
-[company website](https://sigmaprime.io).
-
-## Directory Structure
-
-- [`beacon_node/`](beacon_node/): the "Beacon Node" binary and crates exclusively
-	associated with it.
-- [`docs/`](docs/): documentation related to the repository. This includes contributor
-	guides, etc. (It does not include code documentation, which can be produced with `cargo doc`).
-- [`eth2/`](eth2/): Crates containing common logic across the Lighthouse project. For
-	example: Ethereum 2.0 types ([`BeaconBlock`](eth2/types/src/beacon_block.rs), [`BeaconState`](eth2/types/src/beacon_state.rs), etc) and
-	SimpleSerialize (SSZ).
-- [`protos/`](protos/): protobuf/gRPC definitions that are common across the Lighthouse project.
-- [`validator_client/`](validator_client/): the "Validator Client" binary and crates exclusively
-	associated with it.
-- [`tests/`](tests/): code specific to testing, most notably contains the
-	Ethereum Foundation test vectors.
+\#00](https://lighthouse.sigmaprime.io/update-00.html) blog post or
+[sigmaprime.io](https://sigmaprime.io).
 
 ## Contributing
 
-**Lighthouse welcomes contributors.**
-
-If you are looking to contribute, please head to our
-[onboarding documentation](https://github.com/sigp/lighthouse/blob/master/docs/onboarding.md).
+Lighthouse welcomes contributors.
 
-If you'd like to contribute, try having a look through the [open
-issues](https://github.com/sigp/lighthouse/issues) (tip: look for the [good
-first
-issue](https://github.com/sigp/lighthouse/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-tag) and ping us on the [gitter](https://gitter.im/sigp/lighthouse) channel. We need
-your support!
+If you are looking to contribute, please head to the
+[Contributing](http://lighthouse-book.sigmaprime.io/contributing.html) section
+of the Lighthouse book.
 
 ## Contact
 
 The best place for discussion is the [Lighthouse Discord
 server](https://discord.gg/cyAszAh). Alternatively, you may use the
 [sigp/lighthouse gitter](https://gitter.im/sigp/lighthouse).
 
+Encrypt sensitive messages using our [PGP key](https://keybase.io/sigp).
```

Comment:
Perhaps link to the actual [public key](https://keybase.io/sigp/pgp_keys.asc?fingerprint=dcf37e025d6c9d42ea795b119e7c6cf9988604be)?